lib/ansiblelint/__main__.py

```diff
@@ -154,8 +165,7 @@ def main() -> int:
             print(formatter.format(match))
 
     if matches:
-        hint_about_skips(matches)
-        return 2
+        return report_outcome(matches, options=options)
```

Comment:
https://github.com/ansible/ansible-lint/pull/992 will gate this so it only shows up when NOT quiet, paresable, or parsable_severity are True. Since again this is adding verbose output, I would expect it not to be enabled for those modes

Reply:
It looks like the report_outcome method does the guarding around quiet/parseable itself.